plugins/woocommerce/src/Blocks/BlockTypes/MiniCartFooterBlock.php

```diff
@@ -14,6 +14,16 @@ class MiniCartFooterBlock extends AbstractInnerBlock {
 	 */
 	protected $block_name = 'mini-cart-footer-block';
 
+	/**
+	 * Enqueue frontend data.
+	 *
+	 * @param array $attributes Block attributes.
```

Comment:
Should we duplicate this docs block from `AbstractBlock` instead of writing our own?

Reply:
Oh yea, see d53d6da